package.json

```diff
@@ -14,7 +14,8 @@
   ],
   "scripts": {
     "test": "mocha --inline-diffs --check-leaks --ui tdd --reporter dot --recursive test",
-    "build": "6to5 --source-maps-inline --out-dir dist src",
+    "build": "6to5 --modules commonStandard --source-maps-inline --out-dir dist src",
+    "watch": "6to5 --watch --modules commonStandard --source-maps-inline --out-dir dist src",
     "generate-coverage": "istanbul cover _mocha -- --inline-diffs --check-leaks --ui tdd --reporter dot --recursive test",
```

Comment:
We can use 6to5 for our tests and during coverage now. Maybe in a separate PR though.